docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,40 @@
 # 4.9.0 (Unreleased)
 
+## Overview
 
-## New features
+- [Features and enhancements](#features-and-enhancements)
+    - [Image lazy loading](#image-lazy-loading)
+    - [Other new features](#other-features)
+- [Bugfixes](#bugfixes)
+
+
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy loading {#image-lazy-loading}
```

Comment:
We need a paragraph our two here to explain what lazy loading is, what the benefits are and to link off to relevant Google doc.

We should probably mentioned that this was a partnership with Google as well.